plugins/env/test/controllers/environment_variable_groups_controller_test.rb

```diff
@@ -79,6 +79,27 @@ def self.it_destroys
       end
     end
 
+    describe "a json GET to #index" do
+      it "succeeds" do
+        get :index, format: :json
+        assert_response :success
+        json_response = JSON.parse response.body
+        json_response.keys.must_include "groups"
+      end
```

Comment:
don't need that test ?
... then can simplify to `it "renders json"` without a describe

Reply:
also should be inside of `describe "#index"`